model/common_structures.smithy

```diff
@@ -59,3 +59,28 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+list UserList {
+    member: User
+}
+
+structure User {
+    hash: String,
+    reserved: Boolean,
+    hidden: Boolean,
+    backend_roles: BackendRolesList,
+    attributes: AttributeMap,
+    description: String,
+    opendistro_security_roles: OpendistroSecurityRolesList,
+    static: Boolean
+}
+
+map AttributeMap {
+    key: String,
+    value: String
+}
+
+structure Response{
```

Comment:
May be call this `GenericResponse`? Since it is a common response and can be used by multiple APIs

Reply:
> May be call this `GenericResponse`? Since it is a common response and can be used by multiple APIs

Yeap! I think its best solution for naming